templates/OperationResponseNoBodySnippet.stencil

```diff
@@ -1,16 +1,7 @@
-switch result {
-case let .success(data):
-    {% if op.exceptions.count > 0 %}
-    {% include "OperationResponseDataSnippet.stencil" %}
-    {% endif %}
-    {% include "OperationResponseStatusCodeSnippet.stencil" %}
-    {% include "OperationResponseReturnTypeStatusCodeSnippet.stencil" %}
-    {
-        dispatchQueue.async {
-            completionHandler(.success(()), httpResponse)
-        }
+{% include "OperationResponseStatusCodeSnippet.stencil" %}
+{
+    dispatchQueue.async {
+        completionHandler(.success({% if op.returnType.returnNil  %} nil {% else %} () {% endif %})
```

Comment:
I tried to put this into a prototype of RetrunTypeViewModel, sometthing like
let returnObject = isReturnTypeNilable ? "nil" : "()"
and then in stencil file do
completoinHandler(.success( {{ op.returnType.ReturnObject }})) , httpResponse)

But stencil engine can't resolved `op.returnType.ReturnObject` as long as it has value 'nil'.  So I do above to work around this stencil engine issue.

Reply:
Couldn't you bind it to a string property that's set in the view model as "nil" or "()"?

Reply:
That was the approach i took , create a property in RetrunTypeViewModel with name returnObject and the value of returnObject is either 'nil' or '();. But when render Stencil file stencil , it can't resolve that property for some reason

Reply:
I don't know why Github isn't showing my reply here, but couldn't you have a string property on the view model that is either "nil" or "()"?

Reply:
Per our discussion, let's leave it as but put a comment in the Stencil file. That way we can revisit it in the future.